src/util.ts

```diff
@@ -31,15 +31,6 @@ import * as through from 'through2';
 
 import {Interceptor} from './service-object';
 
-const request = r.defaults({
```

Comment:
How are handling defaults moving forward?

Reply:
They're part of teeny-request.

Reply:
So these specific defaults are part of teeny-request?  Or are we asking each individual library to set these?